### ISSUE TITLE
'truncate-abstract' aria-expanded attribute on wrong element

### ISSUE THREAD

**Issue by jvine** [sul-dlss/earthworks#656]
When truncation is applied, currently the `aria-expanded=true/false` attribute is placed on the `<div>` that contains the content. Seems logical, but it's not the right place: It needs to be on the triggering link or button that expands/collapses the content. 

`<a href="#" data-readmore-toggle="rmjs-1" aria-expanded="false" aria-controls="rmjs-1">Read more</a>`

(Also while you're in there... sentence case preferred on `Read more`.)

_Caveat. The screenreader reads the full text of the abstract regardless of the Read more link; the expand/collapse behaviour is useful is for sighted users only. We can't use aria-hidden, because the link needs keyboard support and that will create a different violation. Suggest we just make it structurally accurate for now._

**Comment by jkeck** [comment]
FWIW, I looked at this briefly, and if it helps anybody look further into this, this is all coming from the `Readmore-js` library.  I haven't seen if they've updated this in their library, but I _think_ where the aria attributes land on the dom/buttons is going to be in their codebase and not in anything that we control.
